custom_components/tuya_local/devices/loratap_zigbee_six_switch.yaml

```diff
@@ -0,0 +1,232 @@
+name: Loratap Six Gang ZigBee Remote Switch
+products: 
+  - id: iszegwpd
+primary_entity:
+  entity: binary_sensor
```

Comment:
When I saw the issue you filed for this, I thought this device might be a candidate for the new Event platform in HA 2023.8.  That would be more work to add though.

Reply:
Wouldn't it be better to do something to make it easier to configure in HA new devices?
After entering the keys to the cloud, it would be known what the devices are under the Tuya Gateways and the product_id would be known to choose the right model.